soco/core.py

```diff
@@ -134,6 +135,7 @@ class SoCo(_SocoSingletonBase):
         get_composers -- Get composers from the music library
         get_tracks -- Get tracks from the music library
         get_playlists -- Get playlists from the music library
+        get_sonos_playlists -- Get Sonos playlists from the music library.
```

Comment:
It is not getting playlists from the music library (which, strictly, is the locally stored music, i.e. not Spotify etc).